client/src/components/getCompareListsData.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from "prop-types";
+
+function getCompareListsData({
+  supermarkets,
+  listData,
```

Comment:
thought listData is an array, maybe an other name would be better?

Reply:
Hey, thank you very much for your input. I adapted your changes. 👍

Reply:
👍  Thanks. I adapted it to groceryList as that´s the content